pheme/transformation/scanreport/gvmd.py

```diff
@@ -0,0 +1,195 @@
+from functools import reduce
+from typing import Callable, Optional, Dict
+from .model import (
+    Count,
+    Filtered,
+    HostResult,
+    HostResults,
+    HostSpecific,
+    NVTResult,
+    QOD,
+    Ref,
+    Report,
+    ResultCount,
+    Results,
+    Solution,
+    Target,
+    Task,
+    Version,
+)
+
+
+def group_by_host(first, second):
```

Comment:
The typings should be
```suggestion
def group_by_host(first: Dict[str, HostResults], second: Dict[str, str]):
```